docs/api-reference/Bucket.md

```diff
@@ -1,3 +1,4 @@
+<!-- markdownlint-disable -->
```

Comment:
docs/api-reference/*.md are generated from Python sources. Maybe these files could be ignored through markdownlint's config?